content/guides/asset-management.md

```diff
@@ -4,116 +4,372 @@ sort: 3
+    output: {
+      filename: 'bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    },
+    module: {
+      rules: [
+        {
+          test: /\.css$/,
+          use: [
+            'style-loader',
+            'css-loader'
+          ]
+        },
+        {
+          test: /\.(png|svg|jpg|gif)$/,
+          use: [
+            'file-loader'
+          ]
+        },
++       {
++         test: /\.(woff|woff2|eot|ttf|otf)$/,
++         use: [
++           'file-loader'
++         ]
++       }
+      ]
+    }
+  };
+
 ```
 
-With this in place you can define a font like so...
+Add some font files to your project:
 
-``` css
-@font-face {
-  font-family: 'MyFont';
-  src:  url('./font.woff2') format('woff2'),
-        url('./font.woff') format('woff');
-  font-weight: 600;
-  font-style: normal;
-}
+__project__
+
+
+``` diff
+  webpack-demo
+  |- package.json
+  |- webpack.config.js
+  |- /dist
+    |- bundle.js
+    |- index.html
+  |- /src
++   |- my-font.woff
++   |- my-font.woff2
+    |- icon.png
+    |- style.css
+    |- index.js
+  |- /node_modules
+```
+
+With the loader configured and fonts in place, you can use incorporate them via an `@font-face` declaration. The local `url(...)` directive will be picked up by webpack just as it was with the image:
+
+__src/style.css__
+
+``` diff
++ @font-face {
++   font-family: 'MyFont';
++   src:  url('./my-font.woff2') format('woff2'),
++         url('./my-font.woff') format('woff');
++   font-weight: 600;
++   font-style: normal;
+  }
+
+  .hello {
+    color: red;
++   font-family: 'MyFont';
+    background: url('./icon.png');
+  }
+```
+
+Now run a new build and let's see if webpack handled our fonts:
+
+``` bash
+npm run build
+
+Hash: b4aef94169088c79ed1c
+Version: webpack 2.6.1
+Time: 775ms
+                                Asset     Size  Chunks                    Chunk Names
+ 5c999da72346a995e7e2718865d019c8.png  11.3 kB          [emitted]
+11aebbbd407bcc3ab1e914ca0238d24d.woff   221 kB          [emitted]
+                            bundle.js   561 kB       0  [emitted]  [big]  main
+   [0] ./src/icon.png 82 bytes {0} [built]
+   [1] ./~/lodash/lodash.js 540 kB {0} [built]
+   [2] ./src/style.css 1 kB {0} [built]
+   [3] ./~/css-loader!./src/style.css 420 bytes {0} [built]
+   [4] ./~/css-loader/lib/css-base.js 2.26 kB {0} [built]
+   [5] ./src/MyFont.woff 83 bytes {0} [built]
+   [6] ./~/style-loader/lib/addStyles.js 8.7 kB {0} [built]
+   [7] ./~/style-loader/lib/urls.js 3.01 kB {0} [built]
+   [8] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [9] (webpack)/buildin/module.js 517 bytes {0} [built]
+  [10] ./src/index.js 503 bytes {0} [built]
 ```
 
-and the relative paths (e.g. `'./font.woff2'`) will be replaced with the final path/filename in your build directory.
+Open up `index.html` again and see if our `Hello webpack` text has changed to the new font. If all is well, you should see the changes. Awesome! This is really starting to come together now, isn't it?
```

Comment:
I think you can probably drop the last line here:

`Awesome! This is really starting to come together now, isn't it?`

or at least the `Awesome!` part, but I'll leave that up to you.